src/parser.cpp

```diff
@@ -785,7 +785,14 @@ void Parser::ParserImpl::loadConnection(const ModelPtr &model, const XmlNodePtr
 
     if (!childNode) {
         IssuePtr issue = Issue::create();
-        issue->setDescription("Connection in model '" + model->name() + "' does not contain any 'map_variables' elements.");
+        std::string des = "Connection in model '" + model->name() + "'";
+        if (connectionId.empty()) {
+            des += " does not contain any 'map_variables' elements. It will not be loaded into the model.";
+        } else {
+            des += " has an id of '" + connectionId + "' but does not contain any 'map_variables' elements.";
+            des += " The connection will not be loaded into the model, and the associated id bookmark will be lost.";
```

Comment:
I wouldn't use the word bookmark. It is an id, that is all.

Reply:
actually, I'd probably stop with `The connection will not be loaded into the model.`

Reply:
or maybe `The connection will be disregarded.`

Reply:
Yesterday @hsorby wanted me to add the note for items with an id, that that info will be lost, on top of the normal empty warning?

Reply:
To me, that information is clearly implied by saying the connection will be disregarded/not loaded.

Reply:
Compromise in cc44b080